sig-multicluster/management-cluster-position-statement.md

```diff
@@ -0,0 +1,46 @@
+# Management Cluster - SIG Multicluster Position Statement
```

Comment:
OFFICIAL POLL for name of the "central" cluster. Please ThumbsUp emoji the chosen name. Feel free to add comments of support one way or another.

Reply:
Management Cluster

Reply:
Hub Cluster

Reply:
Command Cluster

Reply:
Poll is now closed; Hub won with 38 votes, ahead of the 30votes for Management. I'm now renaming the cluster in this PR.

Reply:
Very sorry for missing the POLL :(
I can't see the full list of voters but seems no guys from the Karmada community get involved.